fairmq/Transports.h

```diff
@@ -37,25 +53,15 @@ static std::unordered_map<std::string, Transport> TransportTypes {
     { "ofi", Transport::OFI }
 };
 
-}
-
-namespace fair
-{
-namespace mq
-{
-
-using Transport = ::FairMQ::Transport;
-using ::FairMQ::TransportTypes;
+static std::unordered_map<Transport, std::string> TransportNames {
+    { Transport::DEFAULT, "default" },
```

Comment:
Why do we need `Transport::DEFAULT` again?

Reply:
As discussed, let's keep it like this and remove the `Transport::DEFAULT` entry, once we resolve #4.